systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeScramShaST.java

```diff
@@ -129,6 +129,59 @@ void testReceiveSimpleMessageTlsScramSha() throws Exception {
         assertThat(deleteConsumer(bridgeHost, bridgePort, groupId, name), is(true));
     }
 
+    @Test
+    void testScramShaAuthWithWeirdNamedUser() throws Exception {
```

Comment:
actually it's not just SCRAM-SHA but you are using TLS as well, right? Should we have a test where just SASL is used instead of SALS_SSL?

Reply:
@ppatierno because we using TLS and SCRAM-SHA in whole `HttpBridgeScramShaST` i let it like that. We can add test with `SASL_PLAINTEXT`